crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -470,6 +472,118 @@ unsafe impl QueryData for EntityLocation {
 /// SAFETY: access is read only
 unsafe impl ReadOnlyQueryData for EntityLocation {}
 
+/// The `SpawnedTick` query parameter fetches the [`Tick`] the entity was spawned at.
+///
+/// To evaluate whether the spawn happened since the last time the system ran, the system
+/// param [`SystemChangeTick`](bevy_ecs::system::SystemChangeTick) needs to be used.
+///
+/// If the query should filter for spawned entities instead, use the
+/// [`Spawned`](bevy_ecs::query::Spawned) query filter instead.
+///
+/// # Examples
+///
+/// ```
+/// # use bevy_ecs::component::Component;
+/// # use bevy_ecs::entity::Entity;
+/// # use bevy_ecs::system::Query;
+/// # use bevy_ecs::system::SystemChangeTick;
+/// # use bevy_ecs::query::Spawned;
+/// # use bevy_ecs::query::SpawnedTick;
+/// #
+/// # #[derive(Component, Debug)]
+/// # struct Name {};
+///
+/// fn print_spawn_ticks(query: Query<(Entity, SpawnedTick)>, system_ticks: SystemChangeTick) {
+///     for (entity, spawned) in &query {
+///         if spawned.is_newer_than(system_ticks.last_run(), system_ticks.this_run()) {
+///             print!("new ");
+///         }
+///         println!("entity {entity:?} spawned at {spawned:?}");        
+///     }
+/// }
+///
+/// # bevy_ecs::system::assert_is_system(print_spawn_ticks);
+/// ```
+#[derive(Clone, Copy)]
+pub struct SpawnedTick(());
```

Comment:
Same thing here. Why not this?
```suggestion
pub struct SpawnedTick;
```